### ISSUE TITLE
Not confirming deletion of folders in the trash

### ISSUE THREAD

**Issue by Duncan-Moore** [caligraf/ConfirmBeforeDelete#19]
ConfirmBeforeDelete does not confirm deletion of folders in the trash.

**Comment by caligraf** [comment]
How you do you delete the folders (context menu, del, edit menu ... )?

**Comment by caligraf** [comment]
I have created a new version 1.9.0 where I fixed not showed confirmation messages.
Could you test it ?
https://github.com/caligraf/ConfirmBeforeDelete/releases/download/1.9.0/ConfirmBeforeDelete.xpi

**Comment by Duncan-Moore** [comment]
Many thanks.
Yes, it now gives a confirmation message when deleting a folder in the trash.
A couple of observations:
1) when deleting a folder already in the trash the confirmation message is a bit misleading:
     "Do you want to move the folder <folder> to the trash ?"
2) the confirmation message typeface has changed from sans serif to serif. I don't know if that's by design or not. I prefer the sans serif, but that might just be me.

**Comment by Duncan-Moore** [comment]
I've noticed some odd behaviour with CBD 191:
1) To drag a message to trash, it must be selected before I drag it.
   If it wasn't selected beforehand, then the drag is ignored, or the message that /was/ selected is (wrongly) deleted.
2) Dragging a message to a folder within trash moves the message to trash instead, not the subfolder.

**Comment by caligraf** [comment]
I have done a new version 1.9.2, 
I have fixed the misleadingmessage and the drag of a message in a subfolder of the trash
https://github.com/caligraf/ConfirmBeforeDelete/releases/download/1.9.2/ConfirmBeforeDelete.xpi

I will do another version to fix the drag in trash of a not selected message or a not selected folder.

**Comment by Duncan-Moore** [comment]
Yes, the misleading warning and the drag to a trash subfolder are working for me too in 1.9.2.
Thanks.

**Comment by caligraf** [comment]
i fixed the drag and drop of a message in trash that is not selected in new version 1.9.3
https://github.com/caligraf/ConfirmBeforeDelete/releases/download/1.9.3/ConfirmBeforeDelete.xpi

**Comment by Duncan-Moore** [comment]
Yes, that working for me too with 1.9.3.
Many thanks for all the effort you've put into this.
